include/sbg_driver/message_wrapper.h

```diff
@@ -402,11 +421,27 @@ class MessageWrapper
   /*!
    * Create a ROS standard TwistStamped message from SBG messages.
    * 
-   * \param[in] ref_sbg_imu_msg     SBG-ROS IMU message.
-   * \param[in] ref_p_sbg_imu_msg   SBG-ROS IMU previous message.
-   * \return                        ROS standard TwistStamped message.
+   * \template  T                   	Euler or Quat message.
+   * \param[in] ref_sbg_ekf_nav_msg	SBG-ROS Ekf Nav message.
+   * \param[in] ref_sbg_imu_msg		SBG-ROS IMU message.
+   * \return                        	ROS standard TwistStamped message.
    */
-  const geometry_msgs::TwistStamped createRosTwistStampedMessage(const sbg_driver::SbgImuData& ref_sbg_imu_msg, const sbg_driver::SbgImuData& ref_p_sbg_imu_msg) const;
+  template <typename T>
+  const geometry_msgs::TwistStamped createRosTwistStampedMessage(const T& ref_sbg_ekf_vel_msg, const sbg_driver::SbgEkfNav& ref_sbg_ekf_nav_msg, const sbg_driver::SbgImuData& ref_sbg_imu_msg) const
+  {
+    geometry_msgs::TwistStamped twist_stamped_message;
+  
+    twist_stamped_message.header        = createRosHeader(ref_sbg_imu_msg.time_stamp);
+    twist_stamped_message.twist.angular = ref_sbg_imu_msg.gyro;
+
+    const sbg::SbgMatrix3<float> tdcm = getTransposedDcm(ref_sbg_ekf_vel_msg);
+
+    twist_stamped_message.twist.linear.x = (ref_sbg_ekf_nav_msg.velocity.x * tdcm(0, 0)) + (ref_sbg_ekf_nav_msg.velocity.y * tdcm(0, 1)) + (ref_sbg_ekf_nav_msg.velocity.z * tdcm(0, 2)); 
```

Comment:
Same here multiplying a vector by a matrix should be directly in the Matrix object